install/setup/setup-wizard.sh

```diff
@@ -50,14 +50,19 @@ if [ "$install_db" == "y"  ] || [ "$install_db" == "Y" ] ; then
 
     # Only supporting MySql at this time
     database_type=0
-    while [[ ! ${database_type} =~ ^[1-2]{1}$ ]]; do
+    while [[ ! ${database_type} =~ ^[1-3]{1}$ ]]; do
         echo "Which database (Enter the number)?"
         echo "1) MySQL"
         echo "2) PostgresSQL"
+        echo "3) SQLServer"
         read -p "> " database_type;
     done
 
     while : ; do
+        if [ "$database_type" == "3"  ] ; then
+            echo "Please make sure that the sqlcmd tool is installed."
```

Comment:
Sure. Will be done. I think I will also add a requirement for sqlcmd to be in the PATH - right now it uses default install location.